src/engine/SpatialJoinAlgorithms.h

```diff
@@ -14,6 +26,16 @@ class SpatialJoinAlgorithms {
   Result S2geometryAlgorithm();
   Result BoundingBoxAlgorithm();
 
+  std::vector<box> OnlyForTestingWrapperComputeBoundingBox(
+      const point& startPoint) const {
+    return computeBoundingBox(startPoint);
+  }
+
+  bool OnlyForTestingWrapperContainedInBoundingBoxes(
+      const std::vector<box>& bbox, point point1) const {
+    return containedInBoundingBoxes(bbox, point1);
+  }
```

Comment:
i want the names to convey that these methods are not supposed to get called by the user of the class and that he does not need to care what those methods do. Otherwise i could just make the two wrapped functions public and completely avoid the wrapping altogether. Don't you think it is confusing for users of the class?